frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,216 @@
+<template>
+  <v-skeleton-loader :loading="loading" type="table-tbody">
+    <v-card class="my-4 pa-4 w-100" variant="outlined">
+      <v-container fluid class="pa-0">
+        <AppLabel>Not for Profit Information</AppLabel>
+        <v-card variant="outlined" class="card-outline px-2 py-6 my-4">
+          <v-row no-gutters>
+            <v-col>
+              <AppLabel>Date of Incorporation</AppLabel>
+            </v-col>
+          </v-row>
+          <AppDateInput id="date-to" v-model="model.dateOfIncorporation" :rules="[...rules.required, rules.MMDDYYYY]" :max="todayDate" :disabled="false" label="Select The Date" class="mt-6 mx-6" />
+        </v-card>
+        <v-card v-if="isReferenceLetterRequired" variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+          <v-row no-gutters>
+            <v-col cols="7" md="3" lg="2">
+              <AppLabel>Upload Documents</AppLabel>
+            </v-col>
+            <v-col>
+              <v-tooltip
+                content-class="tooltip"
+                text="A letter from a community representative or agency that has knowledge about the sector and can demonstrate how the society provides services to support the local community."
+                max-width="300px">
+                <template #activator="{ props }">
+                  <v-icon size="large" v-bind="props">mdi-information-slab-circle-outline</v-icon>
+                </template>
+              </v-tooltip>
+            </v-col>
+          </v-row>
+          <v-card variant="outlined" class="card-outline px-2 py-6 my-4 w-100">
+            <AppDocumentUpload
+              v-model="model.documentsToUpload"
+              :uploaded-documents="model.uploadedDocuments"
+              :document-type="DOCUMENT_TYPES.COMMUNITY_LETTER"
+              :document-label="'Letter of Reference for Community Support'"
```

Comment:
can we add this text to DOCUMENT_LABELS in constants.js?

Reply:
done